Dockerfile

```diff
@@ -0,0 +1,31 @@
+ARG NODE_VERSION=8.9.4
+FROM node:$NODE_VERSION
```

Comment:
Most of what this Dockerfile is doing is implementing the Linux instructions from the electron-builder documentation. What do you think about basing this docker image on one of the [electronuserland/builder](https://hub.docker.com/r/electronuserland/builder/) base images, e.g. `FROM electronuserland/builder:8` for Node 8, instead?

Reply:
Ha, should have checked for an electron-specific base image! Yes, I don't mind switching to that if it keeps us lean. @msheiny?

Reply:
:heart:

Reply:
hey @garrettr what are your thoughts on yarn? is it a "drop-in" replacement for npm? i am reading into that electronuserland builder tooling around their docker images.. they are :hankey: 'ing on npm every chance they get and have `yarn` in all the examples.